### ISSUE TITLE
Why just show global_step/sec&examples/sec by pretrain?

### ISSUE THREAD

**Issue by franklee24** [google-research/bert#1258]
I used google colab to pretrain my own data by BERT,but it confuse me that it just show global_step/sec&examples/sec,no loss or others.How can i change my code to fix it?``
`W0825 13:34:11.809314 140652181628800 basic_session_run_hooks.py:724] It seems that global step (tf.train.get_global_step) has not been increased. Current value (could be stable): 12 vs previous value: 12. You could increase the global step by passing tf.train.get_global_step() to Optimizer.apply_gradients or Optimizer.minimize

I0826 00:45:28.850609 140375389116288 session_manager.py:500] Running local_init_op.
INFO:tensorflow:Done running local_init_op.
I0826 00:45:28.996295 140375389116288 session_manager.py:502] Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into /content/drive/MyDrive/colab/bertmaster/bert-master/tmp/pretraining_output/model.ckpt.
I0826 00:45:36.375832 140375389116288 basic_session_run_hooks.py:606] Saving checkpoints for 0 into /content/drive/MyDrive/colab/bertmaster/bert-master/tmp/pretraining_output/model.ckpt.
2021-08-26 00:45:49.021796: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
INFO:tensorflow:global_step/sec: 0.129691
I0826 00:45:58.006054 140375389116288 tpu_estimator.py:2159] global_step/sec: 0.129691
INFO:tensorflow:examples/sec: 4.15011
I0826 00:45:58.006821 140375389116288 tpu_estimator.py:2160] examples/sec: 4.15011
INFO:tensorflow:global_step/sec: 2.06314
I0826 00:45:58.490685 140375389116288 tpu_estimator.py:2159] global_step/sec: 2.06314
INFO:tensorflow:examples/sec: 66.0206
I0826 00:45:58.491152 140375389116288 tpu_estimator.py:2160] examples/sec: 66.0206
INFO:tensorflow:global_step/sec: 2.06575
I0826 00:45:58.974765 140375389116288 tpu_estimator.py:2159] global_step/sec: 2.06575
INFO:tensorflow:examples/sec: 66.1041
I0826 00:45:58.975437 140375389116288 tpu_estimator.py:2160] examples/sec: 66.1041
INFO:tensorflow:global_step/sec: 2.0621
I0826 00:45:59.459806 140375389116288 tpu_estimator.py:2159] global_step/sec: 2.0621
INFO:tensorflow:examples/sec: 65.9873
I0826 00:45:59.460407 140375389116288 tpu_estimator.py:2160] examples/sec: 65.9873
INFO:tensorflow:global_step/sec: 2.06622
I0826 00:45:59.943793 140375389116288 tpu_estimator.py:2159] global_step/sec: 2.06622
INFO:tensorflow:examples/sec: 66.1189
I0826 00:45:59.944195 140375389116288 tpu_estimator.py:2160] examples/sec: 66.1189
INFO:tensorflow:global_step/sec: 2.06073
I0826 00:46:00.429032 140375389116288 tpu_estimator.py:2159] global_step/sec: 2.06073
INFO:tensorflow:examples/sec: 65.9434
I0826 00:46:00.429662 140375389116288 tpu_estimator.py:2160] examples/sec: 65.9434
INFO:tensorflow:global_step/sec: 2.06349
I0826 00:46:00.913643 140375389116288 tpu_estimator.py:2159] global_step/sec: 2.06349
INFO:tensorflow:examples/sec: 66.0316
I0826 00:46:00.913931 140375389116288 tpu_estimator.py:2160] examples/sec: 66.0316
INFO:tensorflow:global_step/sec: 2.06284
I0826 00:46:01.398393 140375389116288 tpu_estimator.py:2159] global_step/sec: 2.06284
INFO:tensorflow:examples/sec: 66.011
I0826 00:46:01.398838 140375389116288 tpu_estimator.py:2160] examples/sec: 66.011
`
<img width="709" alt="0d67ceca1a33ab993200f4bc4f3399b" src="https://user-images.githubusercontent.com/18497603/130883373-e6766abe-8fb4-4eb3-8f3d-75ddfb7debb4.png">
